### PR TITLE
Modify input mask

### PR DESCRIPTION
**Pull request by garrettblehm**
Added blur event on input mask and unit test

### CODE REVIEW

**Review by someyoungideas — APPROVED** [review]
@GarrettB71 After this merges shoot me a message and discuss updating the version for this repo.

File: src/input-mask.test.ts [hunk 1]
```diff
@@ -56,6 +56,19 @@ QUnit.test('Should mask telephone input and strip letters from entry', assert =>
   assert.equal(inputElement.value, '(123) 4  -    ');
 });
 
+QUnit.test('Should mask telephone input on blur', assert => {
```
Comment (samdotme) [inline src/input-mask.test.ts:59]: @GarrettB71 were you able to get this test to pass? The utilities have been updated to not use underscores in the phone number mask.

File: src/input-mask.ts [hunk 2]
```diff
@@ -123,4 +127,18 @@ export class InputMask {
     else
       return returnValue;
   }
+  
+    private static NumberIsTooLong(digitsArray, inputValue): boolean {
```
Comment (samdotme) [inline src/input-mask.ts]: I think this one got tabbed in accidentally.

**Review by samdotme — CHANGES_REQUESTED** [review]
Great job Garrett! Let me know if you need any help with getting Karma to run on this Github repo.

File: src/input-mask.test.ts (hunk 1 above)
Comment (garrettblehm) [reply src/input-mask.test.ts:59]: Fixed!

File: src/input-mask.test.ts [hunk 3]
```diff
@@ -56,6 +56,19 @@ QUnit.test('Should mask telephone input and strip letters from entry', assert =>
...
+  InputMask.TelephoneInputMask(inputElement);
+  inputElement.dispatchEvent(TestUtilities.FocusEvent);
+  assert.equal(inputElement.value, '(   )    -    ');
+
+  inputElement.value = '(1234567890)   -    ';
+  inputElement.dispatchEvent(TestUtilities.BlurEvent);
+
+  window.requestAnimationFrame(() => {
+    assert.equal(inputElement.value, '(123)456-7890');
```
Comment (samdotme) [inline src/input-mask.test.ts]: @GarrettB71 this test should not pass. Unless my previous code is wrong, in which case we should fix that anyway. The value should be `(123) 456-7890`, note the space between the area code and the phone number.

**Review by samdotme — COMMENTED** [review]
Overall, looks great. If you can just look into why that one test passed when it shouldn't have I think we'll be good after that.

**Review by samdotme — APPROVED** [review]
Looks good!

**Pull request merged**